easypost/version.py

```diff
@@ -1,6 +1,8 @@
 VERSION = "6.0.0"
 
-if "-" in VERSION:
-    VERSION_INFO = tuple([int(v) for v in VERSION.split("-")[0].split(".")] + VERSION.split("-")[1:])
-else:
-    VERSION_INFO = tuple(int(v) for v in VERSION.split("."))
+elems = VERSION.split("-")
+nums = [str(v) for v in elems[0].split(".")]
```

Comment:
Explicit over implicit - by doing so the reader (especially while skimming) has to do much less mental work to determine what was meant here and filter our any possibilities of what it could have meant.

```suggestion
elements = VERSION.split("-")
numbers = [str(v) for v in elems[0].split(".")]
```